src/interfaces/external/Convex/Convex-Frax-Platform/IProxyVault.nc

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is this file needed anymore?

Reply:
I have removed Convex-Frax related work to a separate branch branched off of the `dev/macro-audit-14` branch.